converter/internal/otelcolconvert/testdata/loadbalancing.river

```diff
@@ -0,0 +1,24 @@
+otelcol.receiver.otlp "default" {
+	grpc { }
+
+	output {
+		metrics = [otelcol.exporter.loadbalancing.default.input]
+		logs    = [otelcol.exporter.loadbalancing.default.input]
+		traces  = [otelcol.exporter.loadbalancing.default.input]
+	}
+}
+
+otelcol.exporter.loadbalancing "default" {
+	protocol {
+		otlp {
+			client { }
```

Comment:
This surprised me at first, but it's appearing here because the client block is marked as required in Flow.